lib/ui/views/main/main_viewmodel.dart

```diff
@@ -52,4 +53,33 @@ class MainViewModel extends MultipleStreamViewModel {
     _logger.i('argument: NONE');
     _navigationService.navigateToTransactionDetailView();
   }
+
```

Comment:
I think, we could just time block on making the `_snackbarService.showCustomSnackBar` work. If we really can't find a solution we can just settle with this approach. But I would suggest to make the UI to look similar with other snackbars. (For example when adding a transaction with `0` amount)

Reply:
i will look into doing this, and yeah i've seen the other snackbars, will test around and see what can be done! cheers!